Exercise_D and E/index.html

```diff
@@ -1,16 +1,84 @@
 <!DOCTYPE html>
-<html class="no-js">
+<html>
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
+    <title>Page Title</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="./assets/style/index.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      media="screen"
+      href="/Exercise_D and E/assets/style/index.css"
+    />
   </head>
   <body>
-    <!-- CODE HERE -->
+    <!-- header goes here -->
+    <section class="header-container">
+      <div class="header">
+        <img src="/Exercise_D and E/assets/images/logo.png" alt="logo" />
+      </div>
+    </section>
 
-    <script src="" async defer></script>
+    <!-- main content goes here -->
+    <section class="main-container">
+      <div class="content">
+        <div class="text">
+          <h1>beautiful flowers</h1>
+          <span><button>learn more</button></span>
+        </div>
+      </div>
+    </section>
+
+    <!-- footer goes here -->
+    <section class="footer-container">
+      <div class="footer">
+        <!-- for flower and payment -->
+        <div class="row-1">
+          <div class="icon">
+            <img
+              class="icon-size"
+              src="/Exercise_D and E/assets/images/flower-icon.png"
+              alt=""
+            />
+            <p>
+              flowers for every <br />
+              of situation
+            </p>
+          </div>
+          <div class="icon">
+            <img
+              class="icon-size"
+              src="/Exercise_D and E/assets/images/payment-icon.png"
+              alt=""
+            />
+            <p>
+              all major payments <br />
+              kinds accepted
+            </p>
+          </div>
+        </div>
+        <br />
+        <!-- css for phone and truck -->
+        <div class="row-2">
+          <div class="icon">
+            <img
+              class="icon-size"
+              src="/Exercise_D and E/assets/images/phone-icon.png"
+              alt=""
+            />
+            <p>free consultation</p>
+          </div>
+          <div class="icon">
+            <img
+              class="icon-size"
+              src="/Exercise_D and E/assets/images/truck-icon.png"
+              alt=""
+            />
+            <p>city wide delivery</p>
+          </div>
+        </div>
+      </div>
+    </section>
   </body>
 </html>
```

Comment:
You have recreated the website in a beautiful way. Fantastic use of flex. Good work!